File: PassingDataObjC/PassingDataObjC/ViewController.m
//
//  ViewController.m
//  PassingDataObjC
//
//  Created by Field Employee on 3/26/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

#import "ViewController.h"
#import "ViewAController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)onButtonPress:(id)sender {
    ViewAController *vc = [self.storyboard instantiateViewControllerWithIdentifier:@"viewA"];
    
    vc.passText = @"Hello World";
    
    [self presentViewController:vc animated:true completion:nil];
}

@end

File: PassingDataSwift/PassingDataSwift/ViewController.swift
//
//  ViewController.swift
//  PassingDataSwift
//
//  Created by Field Employee on 3/26/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func onPassPropPress(_ sender: Any) {
        let vc = storyboard?.instantiateViewController(withIdentifier: "viewB") as! ViewBController
        vc.passText = "Hello World"
        present(vc, animated: true, completion: nil)
    }
    
}

